cmd/porter/bundle.go

```diff
@@ -73,6 +76,13 @@ func buildBundleBuildCommand(p *porter.Porter) *cobra.Command {
 		"Path to the Porter manifest. Defaults to `porter.yaml` in the current directory.")
 	f.StringVarP(&opts.Dir, "dir", "d", "",
 		"Path to the build context directory where all bundle assets are located.")
+	f.StringVar(&opts.Driver, "driver", porter.BuildDriverDefault,
```

Comment:
I added this flag to build but not publish (for auto-build). I have really mixed feels about the auto-build feature right now. It's super helpful but also makes configuration a bit more of a mess. I was wondering if it should just be a message that you need to rebuild, instead of automatic, just so that we aren't adding flags onto publish that would be confusing and result in making scripts repeat themselves. 🤔

Reply:
I'd be down to create an issue from those thoughts.  Definitely relate to the sentiment that the code complexity around auto-build may be more trouble than it is worth...